tests/unit/torngit/test_github.py

```diff
@@ -262,14 +257,11 @@ async def test_make_http_call_change_host(
         await valid_handler.make_http_call(client, method, url, **query_params)
         assert client.request.call_count == 1
         args, kwargs = client.request.call_args
-        print(args)
-        print(kwargs)
         assert kwargs.get("headers") is not None
         assert kwargs.get("headers").get("Host") == "github.com"
         assert len(args) == 2
         built_url = args[1]
         parsed_url = urlparse(built_url)
-        print(parsed_url)
```

Comment:
There were several print statements in this test file, I removed them, let me know if you want me to add them back.